packages/protocol/deployments/optimism/BorrowingVault-WETHUSDT-1

```diff
@@ -0,0 +1 @@
+0x2EDaEecDA025579be8A0B4E0C77401bB5a56f26A
```

Comment:
no contract at this address

Reply:
Same here, at least it is visible to me on Optimism. 
![image](https://github.com/Fujicracy/fuji-v2/assets/32775237/8179e6f6-74b1-4ac9-8d80-1acb2b760643)